yarn-project/builder/src/contract-interface-gen/typescript.ts

```diff
@@ -379,7 +382,7 @@ export class ${input.name}Contract extends ContractBase {
   ${notesGetter}
 
   /** Type-safe wrappers for the public methods exposed by the contract. */
-  public override methods!: {
+  declare public methods: {
```

Comment:
```suggestion
  public declare methods: {
```
The formatter seems to prefer this version